third_party/blink/renderer/core/loader/resource/font_resource.cc

```diff
@@ -99,16 +99,26 @@ void FontResource::StartLoadLimitTimersIfNecessary(
   DCHECK(!font_load_long_limit_.IsActive());
   load_limit_state_ = LoadLimitState::kUnderLimit;
 
-  font_load_short_limit_ = PostDelayedCancellableTask(
-      *task_runner, FROM_HERE,
-      WTF::BindOnce(&FontResource::FontLoadShortLimitCallback,
-                    WrapWeakPersistent(this)),
-      kFontLoadWaitShort);
-  font_load_long_limit_ = PostDelayedCancellableTask(
-      *task_runner, FROM_HERE,
-      WTF::BindOnce(&FontResource::FontLoadLongLimitCallback,
-                    WrapWeakPersistent(this)),
+  // RUN-1724
+  // [RUN-1457 cleanup]
```

Comment:
Github diff is making this look a lot worse than it is. This is how it looks like for me in VSCode, highlighting the actual differences:
![image](https://user-images.githubusercontent.com/282899/232453327-31b321d5-7150-46ba-b188-919e4e52f690.png)